src/main/resources/config/config.yaml

```diff
@@ -28,7 +28,6 @@ database:
   properties:
     charSet: UTF-8
     hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
-    hibernate.types.print.banner: false
```

Comment:
Why was this removed?

Reply:
I found the banner was still showing up for tests, this PR adds it instead to `src/main/resources/hibernate.properties` which seems to remove it everywhere.